src/openforms/formio/typing/vanilla.py

```diff
@@ -10,6 +10,38 @@ class TextFieldComponent(Component):
     pass
 
 
+class TextAreaComponent(Component):
```

Comment:
What is the advantage of having more specific types than `Component` if they don't add any extra fields?

Reply:
I saw above that we did that for the `TextField` component and I thought I should go in the same direction. I agree with you and I cannot think of any advantages apart from seeing a more specific type in the class definition.